processor/src/main/java/org/jboss/logging/processor/generator/model/ClassModel.java

```diff
@@ -362,4 +368,26 @@ private JExpr determineLocale(final String locale, final JType localeType) {
 
         return initializer;
     }
+
+    private static class JFilerOriginatingElementAware extends JFiler {
```

Comment:
Might be nice to add a little javadoc, or some comments, to explain to other maintainers why this is useful?

Reply:
Thanks for taking a look and for the suggestions 😃🎉 
Applied the changes and pushed an updated version 😃